code/src/components/app-accordion/app-accorion-container/AppAccordionContainer.scss

```diff
@@ -0,0 +1,9 @@
+@import "@design-system";
+
+.spa-accordion-container {
+  &--expanded {
+    margin: 0 !important; //necessary for overriding MUI styles
+  }
+  box-shadow: none;
+  border: 1px solid $ghost-alpha;
```

Comment:
can use our `box-shadow` from design system. can add your property and reuse it
also add variable for border color with `var` like our previous examples of usage

Reply:
fixed

Reply:
Added